docs/analytics/export.md

```diff
@@ -183,15 +183,33 @@ The table below shows the field mapping for the "customDimensions" field.
 
 | Application Insights            | App Center                                     |
 | ------------------------------- | ---------------------------------------------- |
-|  MessageType                    | Type of event (session, event, ...)            |
+|  AppBuild                       | Application build number                       |
 |  AppId                          | App Center App ID                              |
-|  CarrierCountry                 | Carrier Country                                |
-|  TimeZoneOffset                 | Time Zone Offset                               |
-|  CarrierName                    | Carrier Type                                   |
+|  AppNamespace                   | Application namespace                          |
+|  CarrierCountry                 | Carrier country                                |
+|  CarrierName                    | Carrier type                                   |
+|  CountryCode                    | Two-letter country code                        |
+|  EventId                        | App Center Event ID                            |
+|  IngressTimestamp               | Log ingestion timestamp                        |
+|  IsTestMessage                  | Flag indicating if the message is a test one   |
+|  LiveUpdateDeploymentKey        | App Center Code Push deployment key            |
+|  LiveUpdatePackageHash          | App Center Code Push package hash              |
+|  LiveUpdateReleaseLabel         | App Center Code Push release label             |
+|  Locale                         | Device language                                |
+|  MessageId                      | App Center message identifier                  |
```

Comment:
What is that? It's not a sdk field.

Reply:
Must be an internal field. Removing too.